release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md

```diff
@@ -141,11 +141,29 @@ digraph "org/package:0.1.0" {
 
 ### Breaking changes
 
+### Developer tools updates
+
+#### New features
+
+##### OpenAPI Tool
+Added support for allowing resource method capability for the client generation command. 
+Introduced the `--client-methods <resource|remote>` option to select the client method type,  which can be `resource` or `remote`.
+  - `$ bal openapi -i <OpenAPI contract> --client-methods resource`
+  - `$ bal openapi -i <OpenAPI contract> --mode client --client-methods resource`
+
+#### Improvements
+
+##### OpenAPI Tool
+Added support to validate the values that have been assigned to the generated Ballerina types concerning the given OpenAPI schema validation using the API provided by the Ballerina `constraint` package. This validation is available for `int`, `float`, `number`, `string`, and `array`.
```

Comment:
in which openAPI components do we support this? (only in validator or, validator + openAPI to bal generation?)

Reply:
Yeah currently support openapi to ballerina code generation, will mention it here 🙂

Reply:
Address via this commit https://github.com/ballerina-platform/ballerina-release/pull/2058/commits/78f075f7ab1e246060a58b0bde4bf1bb723ac0f3